app/views/modals/edit-resource.html

```diff
@@ -0,0 +1,101 @@
+<div class="modal-resource-action">
```

Comment:
Give a more specific filename than edit-resource.html since this is specific to PVCs.